client/checkout/classic/style.scss

```diff
@@ -39,4 +39,20 @@
 	background-color: #fff;
 }
 
+.wcpay-checkout-email-field {
+	position: relative;
+}
+
+.wcpay-checkout-email-field .wcpay-stripelink-trigger-modal {
+	display: none;
+	position: absolute;
+	right: 5px;
+	top: 26px;
+	width: 64px;
+	height: 40px;
+	background: no-repeat
+		url( '../../../assets/images/payment-methods/link.svg' );
```

Comment:
I think this is the most appropriate location for the Link icon. But there are two similar assets at:
* assets/images/cards/link.png
* client/gateway-icons/link.js

Do you think we should refactor/remove the other ones in a future PR?

Reply:
We could improve that in future PR.

Reply:
I guess we could create a new ticket with the mentioned details. Could be a good first issue for future folks.